prisma/schema.prisma

```diff
@@ -644,6 +662,10 @@ enum ItemActType {
   FEE
 }
 
+enum NonItemActType {
+  NYM_CHANGE
+}
+
```

Comment:
Nice! Probably should've done this with donations!

Reply:
This has already grown far beyond what I originally expected lol